### PR TITLE
Ria 3471 out of time

### PR DESCRIPTION
**Pull request by Lee365**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/RIA-3471

### Change description ###
RIA-3471_Out_Of_Time scenario for F & P

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

**Review by tarun-palisetty — APPROVED** [review]

**Review by tarun-palisetty — COMMENTED** [review]
Any reason for no new/updated functional test

**Review by tarun-palisetty — DISMISSED** [review]
LGTM

File: src/main/java/uk/gov/hmcts/reform/iacaseapi/domain/handlers/postsubmit/AppealPayAndSubmittedConfirmation.java [hunk 1]
```diff
@@ -3,6 +3,7 @@
 import static java.util.Objects.requireNonNull;
```
Comment (Srihari-Nakkala) [inline src/main/java/uk/gov/hmcts/reform/iacaseapi/domain/handlers/postsubmit/AppealPayAndSubmittedConfirmation.java:3]: @Lee365 Will be Good to cover the function Test for the Payment Out of Time appeals

**Review by mkayad — APPROVED** [review]
LGTM

**Review by mkayad — APPROVED** [review]
LGTM

**Review by mkayad — APPROVED** [review]
LGTM

**Review by mkayad — APPROVED** [review]
LGTM

**Review by makowalski — APPROVED** [review]
LGTM

**Pull request merged**